apigee_devportal_kickstart.install

```diff
@@ -150,6 +150,10 @@ function apigee_devportal_setup_monetization(array &$install_state) {
           [ApigeeDevportalKickstartTasksManager::class, 'createStore'],
           [$config['store']],
         ],
+        [
+          [ApigeeDevportalKickstartTasksManager::class, 'createProductType'],
```

Comment:
`ApigeeDevportalKickstartTasksManagerInterface` has a method called `createPaymentGateway`, why is it not being called?

Reply:
I've added back this step, with a the option to disable it in the monetization setup form:
![image](https://user-images.githubusercontent.com/4062676/75493890-cb94f400-596f-11ea-8a9d-ac35d96775ce.png)